src/ExtensionSet.php

```diff
@@ -149,8 +149,6 @@ public function addExtension(ExtensionInterface $extension)
             throw new \LogicException(sprintf('Unable to register extension "%s" as it is already registered.', $class));
         }
 
-        // For BC/FC with namespaced aliases
-        $class = (new \ReflectionClass($class))->name;
```

Comment:
not needed because `$class = \get_class($extension)`, which means aliases are already resolved